pkg/apimachinery/patch/patch.go

```diff
@@ -191,6 +187,18 @@ func (d *PatchSet) Unmarshal(path string, operation *PatchOp, obj any) error {
 	return fmt.Errorf("the path or operation doesn't exist in the patch")
 }
 
+func (p *PatchSet) UnmarshalPatches() ([]string, error) {
```

Comment:
Aha... so this should be IMO the `Unmarshal` as it indeed performs the unmarshal of the whole patchset.